keps/sig-cli/20201022-kubectl-xdg-base-dir.md

```diff
@@ -0,0 +1,212 @@
+---
```

Comment:
Please use the new kep template

Reply:
@wojtek-t believe now I addressed that. Thanks!